docs/DateInput.md

```diff
@@ -5,20 +5,41 @@ title: "The DateInput Component"
 
 # `<DateInput>`
 
-Ideal for editing dates, `<DateInput>` renders an HTML `<input type="date">` element, that most browsers display as a standard [Date Picker](https://mui.com/components/pickers/#date-pickers).
+Ideal for editing dates, `<DateInput>` renders an HTML `<input type="date">` element, that most browsers display as a  [date picker](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date).
 
 ![DateInput](./img/date-input.gif)
 
 The appearance of `<DateInput>` depends on the browser, and falls back to a text input on Safari. The date formatting in this input depends on the user's locale.
 
+## Usage
+
 ```jsx
 import { DateInput } from 'react-admin';
 
 <DateInput source="published_at" />
 ```
 
-`<DateInput>` also accepts the [common input props](./Inputs.md#common-input-props).
+The field value must be a string with the pattern `YYYY-MM-DD` (ISO 8601), e.g. `'2022-04-30'`.
```

Comment:
I believe it also accepts JS `Date` objects, doesn't it? (even though in this case it requires a custom `parse` function)

Reply:
Yes, but I don't see the point in mentioning it, as the data comes from an API response - and therefore was serialized to string. Unless some data providers return dates as `Date` objects?

Reply:
Good point. I don't know any dataProvider that do that, my comment was based on the code only.